source/utilities.tex

```diff
@@ -2969,7 +2969,7 @@
 this indicates that an optional object not containing a value shall be constructed.
 
 \pnum
-Type \tcode{nullopt_t} shall not have a default constructor. It shall be a literal type. Constant \tcode{nullopt} shall be initialized with an argument of literal type.
+Type \tcode{nullopt_t} shall not have a default constructor. It shall be a literal type. The constant \tcode{nullopt} shall be initialized with an argument of literal type.
```

Comment:
No definite article here?

Can you locate the paper that provided the wording, just for double-checking?